classes/PluginInstall.class.php

```diff
@@ -25,15 +25,16 @@ public function styles($hook)
 		if ($hook !== $this->page) {
 			return;
 		}
-		wp_enqueue_style('classicpress-directory-integration-css-plugin', plugins_url('../styles/plugin-page.css', __FILE__), []);
+		wp_enqueue_style('classicpress-directory-integration-css', plugins_url('../styles/plugin-page.css', __FILE__), []);
```

Comment:
In the last PR `-plugin` was added to have an handle that differs from the `-theme` one.

Reply:
Now done.